src/node_crypto.cc

```diff
@@ -226,9 +226,9 @@ static int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
 
 // namespace node::crypto::error
 namespace error {
-void Decorate(Environment* env, Local<Object> obj,
+bool Decorate(Environment* env, Local<Object> obj,
```

Comment:
Yeah, I think this is what I meant by https://github.com/nodejs/node/pull/26868#pullrequestreview-219714840 – thank you :)

I’m personally a fan of using `Maybe<bool>`, like the V8 API does, even if the only possible values are `Just(true)` and `Nothing<bool>()` – it makes it explicit that the return value indicates the presence/non-presence of a pending JS exception.